code/drasil-codeLang/package.yaml

```diff
@@ -0,0 +1,25 @@
+name:                drasil-codeLang
+version:             0.1.0.0
+author:              "Dan Szymczak, Steven Palmer, Jacques Carette, Spencer Smith, Harmanpreet Singh Sagar"
```

Comment:
@JacquesCarette Who should be included in the "author" field of a package's yaml file? (Asking because I don't know and am curious 😅)

Reply:
If you've authored (as in 'created') 20% or more of a file / package, or if your ideas are the reason that package exists.

Some people, like Dan and Steven, shouldn't be authors of brand new packages. Though if it's their code that's moved to a new package, they should be authors.